.changesets/feat_tninesling_coprocessor_cost_access.md

```diff
@@ -0,0 +1,5 @@
+### Move cost context values into DashMap ([PR #5972](https://github.com/apollographql/router/pull/5972))
+
+Allow Rhai scripts and coprocessors to access demand control information via context.
```

Comment:
```suggestion
### Enable router customizations to access demand control info ([PR #5972](https://github.com/apollographql/router/pull/5972))

Rhai scripts and coprocessors can now access demand control information via the context. For more information on Rhai constants to access demand control info, see [available Rhai API constants](https://apollographql.com/docs/router/customizations/rhai-api#available-constants).
```

Reply:
Love this improvement that makes the "How" of this changelog super tangible to the end user. I'll accept this commit and mark this as resolved on behalf of @tninesling.  Thanks, @shorgi!